File: Ejercicios.py
#1,1,2,3,5,8,13
def fibonacci(n): 
    a = 1
    b = 0
    for i in range(n):
        print(str(a)+",",end="")
        a = a + b 
        b = a - b

print(fibonacci(7))

def factorial(num):
    if num == 0 or num == 1:
        return 1
    else:
        return num*factorial(num-1)
    
print(factorial(8))

def palindroma(cadena):
    return cadena == cadena[::-1]

File: 04_Ejercicios/Ejercicios.pl
sub fibonacci{
    my $a = 1;
    my $b = 0;
    my $n = shift;

    for(my $i = 0; $i<$n; $i++){
        print $a;
        print ',';
        $a = $a + $b;
        $b = $a - $b;
    }
    print "\n";

}
  

sub factorial{
    my $n = shift;
    if($n == 0 || $n == 1){
        return 1;
    }else{
        return $n*factorial($n-1);
    }
}

sub palindroma{
    my $string = shift;
    my $string2 = reverse $string;

    if($string eq $string2){
        return 1;
    }else{
        return 0;
    }
}
fibonacci(9);
print factorial(6)."\n";
print palindroma("otto")."\n";